### PR TITLE
Azure AD Connect wizard careful exit

### PR DESCRIPTION
**Pull request by skouti2**
It is better to go back in the wizard, because going forward might accidentally lead to clicking "Configure".

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ In Exchange Management Shell (on-premises) perform the following steps:
 
    3. On the **Connect to Azure AD** screen, enter the appropriate credentials, and then click **Next**. Once connected, keep clicking **Next** until you're on the **Optional Features** screen.
 
-   4. Make sure that **Exchange Mail Public Folders** is not selected. If it isn't selected, you can continue to the next section, *Prerequisite steps in Exchange Online*. If it is selected, click to clear the check box, and then click **Next**.
+   4. Make sure that **Exchange Mail Public Folders** is not selected. If it isn't selected, you can go back in the wizard as long as you can so that you don't make any changes, and then continue to the next section, *Prerequisite steps in Exchange Online*. If it is selected, click to clear the check box, and then click **Next**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md:209]:
```suggestion
   4. Make sure that **Exchange Mail Public Folders** is not selected. If it isn't selected, you can go back in the wizard as long as possible to ensure that you don't make any changes, and then continue to the next section, *Prerequisite steps in Exchange Online*. If it is selected, click to clear the check box, and then click **Next**.
```

**Comment by skouti2** [comment]
Yes, I believe it is correct.

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**